app/components/DR/LocationTracking/NextSteps.js

```diff
@@ -29,7 +30,7 @@ class NextSteps extends Component {
 const styles = StyleSheet.create({
   buttonContainer: {
     marginTop: 24,
-    marginBottom: 24,
+    //marginBottom: 24,
```

Comment:
Remove comment if this line is unnecessary

Reply:
Good good